client/my-sites/backup/wpcom-backup-upsell.tsx

```diff
@@ -50,17 +51,11 @@ const BackupVPActiveBody: FunctionComponent = () => {
 	const onUpgradeClick = useTrackCallback( undefined, 'calypso_jetpack_backup_vaultpress_click' );
 	return (
 		<PromoCard
-			title={ preventWidows( translate( 'Your site has VaultPress' ) ) }
-			image={ { path: JetpackBackupErrorSVG } }
+			title={ preventWidows( translate( 'Your backups are powered by VaultPress' ) ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Backups & Security, Powered by VaultPress' )` *ES Score: 7*